.idea/misc.xml

```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please don't commit files created by pycharm (all files in the folder `.idea`). You can do that with `git rm` command, but be careful to not delete it from your machine, but just in git :) And I guess you will need to do it in the command line, not in pycharm. Check it here: https://stackoverflow.com/a/2047477/1293839